demos/caapf/data/cluster2.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: cluster2
+  namespace: default
+  labels:
+    cni: kindnet
+    env: "dev"
```

Comment:
Just thinking - maybe remove “dev” label to showcase fleet group logic? Because autogenerated `quick-start` group cover both clusters, as well as manually create `dev-group` one.

Reply:
I added this deliberately, the thinking was in the demo:

- Show manually creating a cluster group and how you can match the labels
- Then show that its easier with CAAPF as it removes this step

wdyt?

Reply:
I see, makes sense to me, thanks.